include/unifex/let_error.hpp

```diff
@@ -201,167 +213,198 @@ class _frcvr<Source, Error, Receiver>::type {
   }
 
   const Receiver& get_receiver() const noexcept {
-    UNIFEX_ASSERT(op_ != nullptr);   
+    UNIFEX_ASSERT(op_ != nullptr);
     return op_->receiver_;
   }
 
   operation* op_;
 };
 
-template <typename Source, typename Error, typename Receiver>
-class _op<Source, Error, Receiver>::type {
-  using source_receiver = receiver_type<Source, Error, Receiver>;
-  using final_receiver = final_receiver_type<Source, Error, Receiver>;
+template <typename Source, typename Func, typename Receiver>
+class _op<Source, Func, Receiver>::type {
+  using source_receiver = receiver_type<Source, Func, Receiver>;
+  template <typename Error>
+  using final_receiver = final_receiver_type<Source, Func, Receiver, Error>;
 
 public:
-  template <typename Error2, typename Receiver2>
-  explicit type(Source&& source, Error2&& error, Receiver2&& dest)
-      noexcept(std::is_nothrow_move_constructible_v<Receiver> &&
-               std::is_nothrow_move_constructible_v<Error> &&
-               is_nothrow_connectable_v<Source, source_receiver>)
-  : error_((Error2&&)error)
-  , receiver_((Receiver2&&)dest)
-  {
+  template <typename Func2, typename Receiver2>
+  explicit type(Source&& source, Func2&& func, Receiver2&& dest) noexcept(
+      std::is_nothrow_move_constructible_v<Receiver>&&
+          std::is_nothrow_move_constructible_v<Func>&&
```

Comment:
These feel like the wrong `noexcept` specifications; you're not necessarily move-constructing either a `Receiver` or a `Func` below.